build.gradle

```diff
@@ -197,6 +197,7 @@ task buildUberZip(type: Zip) {
   with uberZipSpec
   destinationDirectory = file("${buildDir}/distributions")
   archiveVersion = project.version
+  reproducibleFileOrder = true
```

Comment:
Is this variable read anywhere?

Reply:
Yes, internally Gradle's `Zip` task will read it via [this](https://docs.gradle.org/current/javadoc/org/gradle/api/tasks/bundling/AbstractArchiveTask.html#isReproducibleFileOrder--).

I think it basically means, "add the files in alphabetical order instead of the order you encounter them in the file system".

Reply:
I think we may need `preserveFileTimestamps = false` as well here to have truly reproducible artifacts.